src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -48,7 +48,10 @@ object AgoraBusiness {
   def importResolver(importString: String) : String = {
     val importStringPattern = """^methods\:\/\/(\S+)\.(\S+).(\d+)""".r
     importString match {
-      case importStringPattern(namespace, name, snapshotId) => AgoraBusiness.findSingle(namespace, name, snapshotId.toInt).getOrElse(new AgoraEntity()).payload.getOrElse("")
+      case importStringPattern(namespace, name, snapshotId) =>
+        //hard coded for tasks...we need to ask wdl parser which type we have
```

Comment:
Ah I think this comment is out of date. I managed to work around the issue by allow a user to specify a Seq of types to search. Previously it had to specify the type in order to search and so we would have had to have know that ahead of time.